default.nix

```diff
@@ -1,6 +1,12 @@
-haskellPackages: rec {
+haskellPackages: nixpkgs: rec {
```

Comment:
Not needed with https://github.com/obsidiansystems/gargoyle/pull/30/files#r322371660